ci/script.sh

```diff
@@ -48,7 +48,7 @@ case $TARGET in
     ;;
 
   clippy)
-    cargo clippy --workspace -- -D warnings -A clippy::unnecessary-cast
+    cargo clippy --workspace -- -D warnings -A clippy::unnecessary-cast -A clippy::bool-to-int-with-if
```

Comment:
Thanks for the question! I've added this allowance rule because clippy was failing in a `paramter_type` `const` that was set to `0` or to `1` based on a boolean condition; Clippy has the opinion that cases like that should be converted from `if cond { 0 } else { 1 }` to `uintT::from(cond)` (see https://github.com/centrifuge/centrifuge-chain/actions/runs/4617361181/jobs/8163526796#step:7:989).

The issue is that we can't do `uint::from` for a `const` AND that if we change the values from 0/1 (for the true/false flows) to, say, 0/2, then this suggestion wouldn't work anymore and we would need to change the code again.

By ignoring this clippy rule we have that code looking the same across runtimes and we won't need to change it if we need to change any of the values. 

Hope that makes sense!

Reply:
Ah ok, makes sense, thanks for the explanation!